### PR TITLE
Add browser support for aria error message for form types

### PR DESCRIPTION
**Pull request by deniselashlley**

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -36,11 +36,6 @@ class Footer extends Component {
         <div className="footer__copyright">
           <p style={styles.p}>{this.copy}</p>
         </div>
-        <div style={styles.footer__branding}>
-          <a title="Comic Relief" href="https://www.comicrelief.com/" rel="noopener noreferrer" target="_blank">
-            <img src="https://www.comicrelief.com/themes/custom/comicrelief/logo.svg" alt="Comic Relief logo" />
-          </a>
-        </div>
       </footer>
```
Comment (AndyEPhipps) [inline src/components/Footer/Footer.js]: Why are you removing the footer here?

**Comment by Heleen-cr** [comment]
could  you add it to the SelectField component too?

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (deniselashlley) [reply src/components/Footer/Footer.js]: not meant to be for this branch

**Pull request merged**